.semaphore/semaphore.yml

```diff
@@ -11,19 +11,19 @@ blocks:
         - name: Java 8 build
           commands:
             - sem-version java 1.8
-            - ./gradlew build
+            - ./gradlew clean check javadoc build

```

Comment:
I don't think clean is needed. The checkout should be clean anyway.

Reply:
I think this can go to juist build. i'll change it back.

Reply:
removed the check javadoc (that's not necessary), kept the clean. Should be clean on checkout, but just to be sure.